### ISSUE TITLE
Barbarian travincal does not work

### ISSUE THREAD

**Issue by ggaydarov** [aeon0/botty#788]
1. Set the bot to work with barbarian
2. Set to run travincal
3. Set it without teleportation
Issues:
1. When bot is running it uses trav wp, but there he can't find the way, to go.
2. Didn't use every time BO and Shout
3. Even when somehow he guesses the way to travincal(with my manual help), he didnt use war cray. He just stays there and hits with autoattack.

version: [Botty v0.7.3]

**Comment by namdnas23** [comment]
Are you using the compiled version or are you development version of botty? Ensure that you have all things set properly within params.ini and if you are using the development version that you have all the assets properly installed. If it can't detect where it is based on assets missing then it will never work properly.

**Comment by ggaydarov** [comment]
> Are you using the compiled version or are you development version of botty? Ensure that you have all things set properly within params.ini and if you are using the development version that you have all the assets properly installed. If it can't detect where it is based on assets missing then it will never work properly.

yes, but I saw the description in params.ini that barb works only with tele, so I guess this issue is not relevent. However could you check the pathfinding for barb tele for travincal, or should I raise another issue for that
